packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -152,6 +159,18 @@ export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderS
     }
   };
 
+  private _onKeyDown = (evt: React.KeyboardEvent<HTMLElement>) => {
+    const { group, onToggleCollapse } = this.props;
+    const { isCollapsed } = group!;
+
+    if ((evt.keyCode === KeyCodes.right && isCollapsed) || (evt.keyCode === KeyCodes.left && !isCollapsed)) {
+      onToggleCollapse!(group!);
```

Comment:
Consider using the '!' operator only when absolutely necessary. For example, calling the onToggleCollapse method only makes sense if you have a valid group object.

if (onToggleCollapse && group) {
   onToggleCollapse(group);

   // we should probably continue bubbling the event if we did not handle it
   evt.stopPropagation();
   evt.preventDefault();
}

Reply:
Usage of `!` should be so scarce that we should leave a comment justifying the usage.